### PR TITLE
Add doc for post with token

### PR DESCRIPTION
**Pull request by gussiciliano**
## SUMMARY
- add doc for post with token

## ISSUE
https://github.com/Wolox/wor-requests/issues/33

### CODE REVIEW

**Comment by gussiciliano** [comment]
@mishuagopian i don't know to who assign this PR, anyway, I can't options to using jaja

**Review by hdf1996 — APPROVED** [review]
:D

**Comment by gussiciliano** [comment]
:gus-zoom-very-happy:

File: README.md [hunk 1]
```diff
@@ -62,6 +62,36 @@ puts GithubService.new.repositories('alebian')
...
+```ruby
+require 'wor/requests'
+
+class ExternalService < Wor::Requests::Base
+  def request_to_post(authorization)
+    post(
+      attempting_to: 'post to an external service',
+      path: '/some_endpoint',
+      headers: {
+        Authorization: authorization
```
Comment (alebian) [inline README.md]: What do you think of adding a 'Content-type' header to the get example above?

File: README.md [hunk 2]
```diff
@@ -62,6 +62,36 @@ puts GithubService.new.repositories('alebian')
 
 Or, even easier, run `rails generate wor:requests:service NAME` in your Rails root
 
+
+If you work with Authorization token you can write something like this:
```
Comment (alebian) [inline README.md]: If you need to send body parameters in a post request..

File: README.md [hunk 3]
```diff
@@ -62,6 +62,36 @@ puts GithubService.new.repositories('alebian')
...
+
+```ruby
+require 'wor/requests'
+
+class ExternalService < Wor::Requests::Base
+  def request_to_post(authorization)
+    post(
+      attempting_to: 'post to an external service',
```
Comment (alebian) [inline README.md]: `'Make a POST request to an external service.'`

File: README.md [hunk 4]
```diff
@@ -62,6 +62,36 @@ puts GithubService.new.repositories('alebian')
...
 Or, even easier, run `rails generate wor:requests:service NAME` in your Rails root
 
+
+If you work with Authorization token you can write something like this:
+
+```ruby
+require 'wor/requests'
+
+class ExternalService < Wor::Requests::Base
+  def request_to_post(authorization)
```
Comment (alebian) [inline README.md]: I'd call this `post_request_example`

**Review by alebian — CHANGES_REQUESTED** [review]

**Comment by gussiciliano** [comment]
@alebian the changes are already :D

**Comment by gussiciliano** [comment]
@alebian remember check this PR :D

**Review by alebian — APPROVED** [review]

**Pull request merged**